addons/mail/static/src/components/discuss_sidebar/discuss_sidebar.xml

```diff
@@ -12,69 +12,123 @@
+                        <div class="o_DiscussSidebar_groupHeaderItem o_DiscussSidebar_groupHeaderItemAdd fa fa-plus" t-on-click="_onClickChatAdd" title="Start a conversation"/>
+                    </div>
+                    <div class="o_DiscussSidebar_list">
+                        <t t-if="discuss.isAddingChat">
+                            <div class="o_DiscussSidebar_item o_DiscussSidebar_itemNew">
+                                <AutocompleteInput
+                                    class="o_DiscussSidebar_itemNewInput"
+                                    isFocusOnMount="true"
+                                    placeholder="'Find or start a conversation...'"
+                                    select="_onAddChatAutocompleteSelect"
+                                    source="_onAddChatAutocompleteSource"
+                                    t-on-o-hide="_onHideAddingItem"
+                                />
+                            </div>
+                        </t>
+                        <t t-foreach="quickSearchPinnedAndOrderedChats" t-as="chat" t-key="chat.localId">
+                            <DiscussSidebarItem
+                                class="o_DiscussSidebar_item"
+                                threadLocalId="chat.localId"
                             />
+                        </t>
+                    </div>
+                </div>
+                <div class="o_DiscussSidebar_group o_DiscussSidebar_groupRoom">
+                    <div class="o_DiscussSidebar_groupHeader">
+                        <div class="o_DiscussSidebar_groupHeaderItem o_DiscussSidebar_groupTitle">
+                            Chat Rooms
                         </div>
-                    </t>
-                    <t t-foreach="quickSearchPinnedAndOrderedChats" t-as="chat" t-key="chat.localId">
-                        <DiscussSidebarItem
-                            class="o_DiscussSidebar_item"
-                            threadLocalId="chat.localId"
-                        />
-                    </t>
+                        <div class="o-autogrow"/>
+                        <div class="o_DiscussSidebar_groupHeaderItem o_DiscussSidebar_groupHeaderItemAdd fa fa-plus"/>
+                    </div>
+                    <div class="o_DiscussSidebar_list">
+                        <t t-foreach="rooms" t-as="room" t-key="room.localId">
+                            <div t-att-class="{
+                                'o_DiscussSidebar_groupRoom_active': env.messaging.chatRoomLocalId === room.localId,
+                            }" class="o_DiscussSidebarItem" t-on-click="_onClickRoom(room.localId)">
+                                <t t-esc="room.name"/>
+                            </div>
+                        </t>
+                    </div>
                 </div>
             </div>
+            <t t-if="env.messaging.chatRoomLocalId">
+                <!-- TODO component? -->
```

Comment:
I think in latest spec this should be part of the systray so that it is avariable on every page, not specific to sidebar in discuss.

So yes definitely component (+ model)